### PR TITLE
Use version of bech32 from hackage

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

**Comment by rvl** [comment]
@hamishmack Would you please be able to review the Haskell.nix update?

File: cabal.project [hunk 1]
```diff
@@ -1,4 +1,4 @@
-index-state: 2020-05-15T00:00:00Z
+index-state: 2020-07-15T00:00:00Z
```
Comment (erikd) [inline cabal.project:1]: Ok, that is brave. I have not been updating that to be any more recent than 2 weeks old.

**Comment by hamishmack** [comment]
> Would you please be able to review the Haskell.nix update?

LGTM

**Comment by dcoutts** [comment]
bors merge

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**